Meta/CMake/flatpak/org.ladybird.Ladybird.json

```
@@ -0,0 +1,416 @@
+    },
+    {
+      "name": "highway",
+        "buildsystem": "cmake-ninja",
+        "sources": [
+          {
+            "type": "git",
+            "url": "https://github.com/google/highway.git",
+            "tag": "1.2.0",
+            "disable-submodules": true
+          }
+        ],
+        "config-opts": [
+          "-DCMAKE_BUILD_TYPE=Release",
+          "-DCMAKE_PREFIX_PATH=/app",
+          "-DHWY_ENABLE_INSTALL=ON",
+          "-DHWY_ENABLE_EXAMPLES=OFF",
+          "-DHWY_ENABLE_TESTS=OFF",
+          "-DHWY_ENABLE_CONTRIB=OFF"
+        ]
+    },
+    {
+      "name": "libjxl",
+        "buildsystem": "cmake-ninja",
+        "sources": [
+          {
+            "type": "git",
+            "url": "https://github.com/libjxl/libjxl.git",
+            "tag": "v0.11.1",
+            "disable-submodules": true
+          }
+        ],
+        "config-opts": [
+          "-DCMAKE_BUILD_TYPE=Release",
+          "-DCMAKE_PREFIX_PATH=/app",
+          "-DJPEGXL_VERSION=0.11.1",
+          "-DJPEGXL_FORCE_SYSTEM_HWY=ON",
+          "-DJPEGXL_FORCE_SYSTEM_BROTLI=ON",
+          "-DJPEGXL_FORCE_SYSTEM_HWY=ON",
+          "-DJPEGXL_FORCE_SYSTEM_LCMS2=ON",
+          "-DJPEGXL_ENABLE_TOOLS=OFF",
+          "-DJPEGXL_ENABLE_BENCHMARK=OFF",
+          "-DJPEGXL_ENABLE_DOXYGEN=OFF",
+          "-DJPEGXL_ENABLE_EXAMPLES=OFF",
+          "-DJPEGXL_ENABLE_FUZZERS=OFF",
+          "-DJPEGXL_ENABLE_JNI=OFF",
+          "-DJPEGXL_ENABLE_MANPAGES=OFF",
+          "-DJPEGXL_ENABLE_OPENEXR=OFF",
+          "-DJPEGXL_ENABLE_PLUGINS=OFF",
+          "-DJPEGXL_ENABLE_SJPEG=OFF",
+          "-DJPEGXL_ENABLE_SKCMS=OFF",
+          "-DJPEGXL_ENABLE_TCMALLOC=OFF",
+          "-DBUILD_TESTING=OFF",
+          "-DCMAKE_FIND_PACKAGE_TARGETS_GLOBAL=ON",
+          "-DJPEGXL_BUNDLE_LIBPNG=OFF",
+          "-DCMAKE_INSTALL_RPATH=\"$ORIGIN:$ORIGIN/../lib64:/app/lib64\""
+        ]
+    },
+    {
+      "name": "skia",
+      "buildsystem": "simple",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://skia.googlesource.com/skia.git",
+          "commit": "chrome/m129"
+        },
+        {
+          "type": "file",
+          "path": "skia-configure.sh"
+        },
+        {
+          "type": "file",
+          "path": "skia-build.sh"
+        },
+        {
+          "type": "file",
+          "path": "skia-install.sh"
+        }
+      ],
+      "build-commands": [
+        "./skia-configure.sh",
+        "./skia-build.sh",
+        "./skia-install.sh"
+      ]
+    },
+    {
+      "name": "libtommath",
+      "buildsystem": "cmake-ninja",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/libtom/libtommath.git",
+          "tag": "v1.3.0"
+        }
+      ],
+      "config-opts": [
+        "-DCMAKE_BUILD_TYPE=Release",
+        "-DCMAKE_PREFIX_PATH=/app",
+        "-DBUILD_SHARED_LIBS=ON"
+      ]
+    },
+    {
+      "name": "Ladybird",
+      "buildsystem": "cmake-ninja",
```

Some files are not installed, e.g. search engine cannot be set in settings.

```suggestion
      "buildsystem": "cmake-ninja",
      "post-install": [
        "install -D Base/res/ladybird/ladybird.css /app/share/Lagom/ladybird/ladybird.css",
        "cp -r Base/res/ladybird/about-pages/settings /app/share/Lagom/ladybird/about-pages"
      ],
```

---

Ah, thanks for the file list there. These should be add to the list of files installed from UI/cmake/ResourceFiles.cmake though, not just here in the flatpack. This implies that `install_ladybird_resources()` is missing a few things.